pkg/reconciler/scheduling/placement/placement_reconcile.go

```diff
@@ -97,18 +77,18 @@ func (c *controller) listLocations(clusterName logicalcluster.Name) ([]*scheduli
 	return ret, nil
 }
 
-func (c *controller) listWorkloadClusters(clusterName logicalcluster.Name) ([]*workloadv1alpha1.WorkloadCluster, error) {
-	items, err := c.workloadClusterIndexer.ByIndex(byWorkspace, clusterName.String())
+func (c *controller) listNamespace(clusterName logicalcluster.Name) ([]*corev1.Namespace, error) {
```

Comment:
```suggestion
func (c *controller) listNamespacesWithAnnotation(clusterName logicalcluster.Name) ([]*corev1.Namespace, error) {
```

Reply:
would rather drop the annotation check here and let just the reconciler do the filtering. Then drop my suggestion here.